### DISCUSSION TITLE
Search function in playlists

### DISCUSSION THREAD

**Discussion by BobWs** [navidrome/navidrome#3059] [Ideas]
Hi,
I would like to request a search function in the playlist section. Take spotify for example, there you have the ability to find songs in a playlist.
Would you consider implementing something similar for ND?

  
![Screen_Shot 3](https://github.com/navidrome/navidrome/assets/5485820/74e0f5f8-4c22-424d-8453-b7707a55ac57)

**Comment by jthwho** [comment]
That'd be awesome.  It'd be great to be able to quickly find a song on the playlist to remove it.

**Comment by deluan** [comment]
Yes, this is in the plans for the new UI.

**Comment by bblater** [comment]
Has this been added? I'm using the latest version 0.60.0 and don't see this ability. thx

**Reply by deluan** [reply]
No, the work on the new UI just started.

**Reply by dweymouth** [reply]
In the meantime, some 3rd party clients offer this capability :)
